src/core/hle/service/service.cpp

```diff
@@ -78,7 +78,7 @@ const std::array<ServiceModuleInfo, 41> service_module_map{
       false},
      {"CECD", 0x00040130'00002602, CECD::InstallInterfaces, false},
      {"CFG", 0x00040130'00001702, CFG::InstallInterfaces, false},
-     {"DLP", 0x00040130'00002802, DLP::InstallInterfaces, false},
+     {"DLP", 0x00040130'00002802, DLP::InstallInterfaces, true},
```

Comment:
This makes the module load when the "enable lle modules for online play" is selected. Is this intended?